wallet/src/de/schildbach/wallet/ui/send/SendCoinsFragment.java

```diff
@@ -1372,9 +1378,13 @@ private void executeDryrun() {
 
             try {
                 // check regular payment
-                final SendRequest sendRequest = createSendRequest(finalPaymentIntent, RequestType.REGULAR_PAYMENT, false);
+                SendRequest sendRequest = createSendRequest(finalPaymentIntent, RequestType.REGULAR_PAYMENT, false, false);
 
                 wallet.completeTx(sendRequest);
+                if(checkDust(sendRequest)) {
+                    sendRequest = createSendRequest(finalPaymentIntent, RequestType.INSTANT_SEND_AUTO_LOCK, false, true);
```

Comment:
Should't be `RequestType.REGULAR_PAYMENT` instead of `RequestType.INSTANT_SEND_AUTO_LOCK` here?

Reply:
Yes, copy/paste error.

Reply:
This was repaired in 15832ce